studyManager.py

```diff
@@ -36,6 +48,27 @@ def main(args):
             studies = add_study(cfg["connection_string"], args.add)
             print(f"Study Id: {studies}")
 
+        # Delete study
+        if args.remove is not None:
+            # delete
+            remove_study(cfg["connection_string"], args.remove)
+
+        # Rename study
+        if args.update:
+            rows = get_studies(cfg["connection_string"])
+            if (len(rows) >= 1):
+                # display the list of studies to the user
+                layout = "{!s:20} {!s:14}"
+                print(layout.format("Study Name", "Study Id"))
+                print("-" * 34)
+                for row in rows:
+                    print(layout.format(*(row[1], row[0])))
+            else:
+                print("Table is empty")
+            id = input("Enter study id:")
+            sname = input("Enter new study name:")
+            rename_studyname(cfg["connection_string"], id, sname)
```

Comment:
Sure, but if everything is is being provided via the command line, then why not just have them supply the parameters there as well. No need to have user interaction if you don't need it.